zboxcore/client/entity.go

```diff
@@ -10,7 +10,7 @@ type SignFunc func(hash string) (string, error)
 
 type Client struct {
 	*zcncrypto.Wallet
-	signatureSchemeString string `json:"signature_scheme"`
```

Comment:
I would suggest to leave `json` there. We don't know if it is used by anyone.

Reply:
No. The field is not exported so it won't be used by json as well